package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A small quote-serving JSON API.",
   "main": "server.js",
   "scripts": {
-    "start": "node server.js"
+    "start": "nodemon server.js"
```

Comment:
@nikhil3113 Thanks so much for your help! I am wondering if we could possibly have 2 scripts to start the program? One for a regular user that runs it without Nodemon and one for a developer that runs with Nodemon? 

I probably should have specified this in the issue but I am still learning.